manage-apps/register.html

```diff
@@ -206,6 +206,10 @@
           $('#sandboxRegistration button[type="submit"]').attr('disabled', true);
         },
         error: function(res) {
+          console.log('******',res)
+          if (res.statusText === 'error') {
```

Comment:
I wonder if there is ever a case where we would end up showing both this error message and the one on line 215? I'm not sure what the error object looks like in a login timeout case.